config/locales/client.en.yml

```diff
@@ -1933,7 +1935,7 @@ en:
         fine_print: "We are asking you to confirm your identity because this is a potentially sensitive action. Once authenticated, you will only be asked to re-authenticate again after a few hours of inactivity."
       password:
         title: "Password"
-        too_short: "Your password is too short."
+        too_short: "Your password is too short (minimum is %{password_min_length} characters)."
```

Comment:
close, but this does not work :) https://meta.discourse.org/discourse-ai/ai-bot/shared-ai-conversations/v9RZbSKUtP02ZoKXqHVAug

Reply:
O good catch. I always forget about this when dealing with counts.

Reply:
Yep, fixing